users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -744,7 +744,7 @@ def run_scf_baseline():
             ),
         },
         num_epochs=450,
-        prefix="conformer_bs2x5k_",
```

Comment:
All batch sizes are moved to 5000 to make all future audio perturbation experiments small enough for 11GB cards